### PR TITLE
Ramico/new good function

### PR DESCRIPTION
**Pull request by ramico**

### CODE REVIEW

File: includes/class-wc-siftscience-events.php [hunk 1]
```diff
@@ -33,7 +33,6 @@ class WC_SiftScience_Events {
 		 * @var WC_SiftScience_Api_Account
 		 */
 		private $account;
-
```
Comment (nabsul) [inline includes/class-wc-siftscience-events.php:36]: undo this change please

File: includes/class-wc-siftscience-events.php [hunk 2]
```diff
@@ -440,8 +448,8 @@ private function send_order_event( $order_id, $type = 'create' ) {
 		 * @param string $order_id The order ID.
 		 */
 		public function update_order_status( $order_id ) {
-			$order_id = wc_get_order( $order_id );
-			$user_id  = $this->options->get_user_id( $order_id );
+			$order = wc_get_order( $order_id );
+			$this->order_status->try_update_order_status( $order, $this->options->get_user_id( $order ) );
```
Comment (nabsul) [inline includes/class-wc-siftscience-events.php:452]: image

The function try_update_order_status() does not take a user-id as its second parameter.

Comment (ramico) [reply includes/class-wc-siftscience-events.php:452]: I didn't catch that ok

File: includes/class-wc-siftscience-events.php [hunk 3]
```diff
@@ -433,6 +441,16 @@ private function send_order_event( $order_id, $type = 'create' ) {
...
 		}
+
+		/**
+		 * Updates order status
+		 *
+		 * @param string $order_id The order ID.
+		 */
+		public function update_order_status( $order_id ) {
+			$order = wc_get_order( $order_id );
+			$this->order_status->try_update_order_status( $order, $this->comm->get_user_score( $this->options->get_user_id( $order ) ) );
```
Comment (nabsul) [inline includes/class-wc-siftscience-events.php:452]: May I suggest rewriting this to:

```
$user_id = $this->options->get_user_id( $order );
$score = $this->comm->get_user_score( $user_id );
$this->order_status->try_update_order_status( $order, $score );
```

It's easier to read that way.

**Review by nabsul — APPROVED** [review]
